### PR TITLE
[IN PROGRES] added a character STAR WAR CAT

### PR DESCRIPTION
**Pull request by gaurang7goel**
- [x] Read and understood [see CONTRIBUTING.md](https://github.com/fossasia/labyrinth/blob/master/CONTRIBUTING.md)
- [x] Included a preview link and screenshot showing after and before the changes.
- [x] Included a description of change below.
- [x] Tried to [squash the commits][squash].
- [x] Added Screenshot to Indicate Location of Change in Game

### Changes done in this Pull Request

- View my work at https://pr-930-fossasia-labyrinth.surge.sh

https://gaurang7goel.github.io/labyrinth/

- Fixes #<Add issue number here>

### The problem I want to solve or the facility I want to improve is/are

I have added a character STAR WAR CAT .
### My steps to solve it

I have added star war cat.svg in characters folder and made code changes in gui.js in js folder.
### Screenshots, Location of Changes

change

[squash]: https://stackoverflow.com/questions/5189560/squash-my-last-x-commits-together-using-git

### CODE REVIEW

File: js/gui.js [hunk 1]
```diff
@@ -71,6 +71,10 @@ function choosePlayerCharacter(player) {
             "characters/ironman.svg": "Iron Man",
             "characters/LittleAlien.png": "Little Alien",
             "characters/barbariansword.svg": "Barbarian Sword",
+            "characters/star war cat.svg": "STAR WAR CAT"
+
+   
+
```
Comment (YashKumarVerma) [inline js/gui.js]: please remove the empty lines

**Comment by NishithP2004** [comment]
Good work

**Comment by gaurang7goel** [comment]
sir pls check now.

**Comment by YashKumarVerma** [comment]
Closed?

**Pull request closed without merging**